styles/global.scss

```diff
@@ -63,3 +65,44 @@ body {
         }
     }
 }
+
+.footer {
+    background: #212121;
+
+    .footer_wrapper {
+        display: flex;
+        justify-content: space-between;
+        padding: 1.5rem 0.3rem;
+        align-items: baseline;
+        max-width: 1240px;
+        margin-inline: auto;
+    }
+    span {
+        font-size: 1rem;
+        display: inline-flex;
+        align-items: center;
+        color: var(--white);
+        i {
+            font-size: inherit;
+            color: var(--red);
+            margin: 0 0.4rem;
+        }
```

Comment:
can use px overe here , because if the values are less than 10 px , you can use px values.

Reply:
i am comfortable with rem 🙂